.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: Found a Bug that needs to be fixed?
+body:
+- type: markdown
+  attributes:
+    value: |-
+      Please join the [Discord Server](https://discord.gg/0hMr4ce0tIl3SLv5) for questions or ask them in [our Discussions](https://github.com/DV8FromTheWorld/JDA/discussions).
+      
+      Keep in mind that this isn't the place to learn Java.  
+      Please head over to [StackOverflow](https://stackoverflow.com/questions/tagged/java) for your general programming questions.
+- type: checkboxes
+  attributes:
+    label: General Troubleshooting
+    description: You confirm to have made the following checks first.
+    options:
+      - label: I have checked for similar issues on the Issue-tracker.
+        required: true
+      - label: I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
+        required: true
+      - label: I have checked the branches or the maintainers' PRs for upcoming bug fixes.
+        required: true
```

Comment:
I'm sorta leaning against doing this because github does the 3/3 tasks nonsense now. Is there a better way to "check" things without needing to put them in the issue?

Reply:
That's pretty much a visual thing. And since all points are required is there no case where you would have like "2 of 3 tasks complete" or something alongside this...